test/test_helper.exs

```diff
@@ -1,2 +1,4 @@
-ExUnit.start
+Application.ensure_all_started(:mimic)
 Application.ensure_all_started(:http_server)
+Mimic.copy(:hackney)
```

Comment:
We can probably leave this here for now but when we do a PR for the main exvcr repository we should move this to another module, to only happen if the used mocking library is mimic.